charts/rancher-logging/104.0.0+up4.4.0/values.yaml

```diff
@@ -362,13 +362,16 @@ additionalLoggingSources:
     fluentbit:
       log_level: "info"
       mem_buffer_limit: "5MB"
+    daemonSetResources: {}
   rke2:
     enabled: false
     stripUnderscores: false
+    daemonSetResources: {}
   k3s:
     enabled: false
     container_engine: "systemd"
     stripUnderscores: false
+    daemonSetResources: {}
```

Comment:
I was unsure what to call them. We could do `resources`, but that is maybe a bit vague. A more specific name (refering to `DaemonSet`) will make others aware of where the resources are actually used.